src/index.js

```diff
@@ -218,25 +217,9 @@ function EvolvClient(opts) {
    */
   this.emit = function(type, metadata, flush) {
     context.pushToArray('events', {type: type,  timestamp: (new Date()).getTime()});
-    eventBeacon.emit(type, assign({
-      uid: context.uid,
-      metadata: metadata
-    }), flush);
```

Comment:
I guess it's a real complaint - this appears to be the last use of `flush` here, so it can be removed from the params

Reply:
Yep -- pushed that change